tests/core_tests/test_includes.py

```diff
@@ -19,7 +19,7 @@
 from mock import patch
 
 
-class TestIncludes(object):
+class Includes(object):
```

Comment:
Changes in this file are to satisfy the pytest runner. Classes starting with Test but with an init method emits warnings.